src/main/java/edu/cornell/kfs/concur/batch/service/impl/ConcurDetailLineGroupForCollector.java

```diff
@@ -320,7 +320,7 @@ protected void addOriginEntriesForCorpCardPersonalExpenseDebitLinesIfNecessary(C
         ConcurStandardAccountingExtractDetailLine firstDebitLine = debitLines.get(0);
         if (cashOffsetAdjustment.isNonZero()) {
             addOriginEntryForCorpCardPersonalExpense(
-                    entryConsumer, firstDebitLine, collectorHelper.getCashOffsetObjectCode(), cashOffsetAdjustment);
+                    entryConsumer, firstDebitLine, collectorHelper.getPaymentOffsetObjectCode(), cashOffsetAdjustment);

```

Comment:
I think it would be a good idea to also rename the "cashOffsetAdjustment" variable here, as well as any other variables and methods in this class that have "cashOffset" in the name.